src/main/java/org/jitsi/impl/reservation/rest/RESTReservations.java

```diff
@@ -217,7 +216,7 @@ else if (result.statusCode == 409
     /**
      * Deletes conference for given <tt>mucRoomName</tt> through the API.
      */
-    Result deleteConference(String mucRoomName)
+    private synchronized Result deleteConference(String mucRoomName)
```

Comment:
If methods are private and run within the synchronised I disagree we want to wrap it again.